src/macros.jl

```diff
@@ -625,12 +625,102 @@ julia> @with df @byrow :x * :y
     global scope, then a variable cannot be assigned without using the `global` keyword.
     If the parent scope is a local scope (inside a function or let block for example),
     the `global` keyword is not needed to assign to that parent scope.
+
+!!! note
```

Comment:
should this be indented by 4 spaces?

Reply:
Nope it works

Reply:
Though I may need you to write the implementation for this in the future since I will have to rely on DataFrames internals to really get this working.

Reply:
So why do you indent `!!! note` above (line 621) with 4 spaces and not here?

Reply:
I see for spaces. Can you add a change via github? (We can also worry about this in another PR)

Reply:
ah - you are right. I have mistakenly mixed this with line 352.